src/MyEventEmitter.js

```diff
@@ -1,12 +1,90 @@
-'use strict';
-
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, callback) {
+    const listeners = this.events[eventName];
+
+    if (listeners) {
+      listeners.push(callback);
+    } else {
+      this.events[eventName] = [callback];
+    }
+
+    return this;
+  }
+  once(eventName, callback) {
+    const listeners = this.events[eventName];
+
+    const onceCallback = () => {
+      callback();
+      this.off(eventName, onceCallback);
+    };
+
+    if (listeners) {
+      listeners.push(onceCallback);
+    } else {
+      this.events[eventName] = [onceCallback];
+    }
+
+    return this;
+  }
+  off(eventName, callback) {
+    const listenerIndex = this.events[eventName].indexOf(callback);
+
+    if (listenerIndex >= 0) {
+      this.events[eventName].splice(listenerIndex, 1);
+    }
+
+    return this;
+  }
+  emit(eventName, ...args) {
```

Comment:
```suggestion
  off(eventName, callback) {
    const listenerIndex = this.events[eventName].indexOf(callback);

    if (listenerIndex >= 0) {
      this.events[eventName].splice(listenerIndex, 1);
    }

    return this;
  }
  
  emit(eventName, ...args) {
```

Don't forget to add empty line between methods